statsmodels/genmod/cov_struct.py

```diff
@@ -583,11 +582,10 @@ def update_nogrid(self, params):
         dep_params /= dn
         self.dep_params = dep_params[1:] / dep_params[0]
 
-
     def covariance_matrix(self, endog_expval, index):
 
         if self.grid:
-            return self.covariance_matrix_grid(endog_expal, index)
+            return self.covariance_matrix_grid(endog_expval, index)
```

Comment:
I added a regression test for the place where the bug was fixed.  We don't have a generic way to run through all of the covariance structures in a test right now, so it's not easy to have simple consistency checks like this.  I will add better, more systematic testing of GEE in a future PR.